jujugui/static/gui/src/app/init.js

```diff
@@ -1104,7 +1104,7 @@ class GUIApp {
     if (!err) {
       return;
     }
-    if (!yui.juju.views.utils.isRedirectError(err)) {
+    if (!viewUtils.isRedirectError(err)) {
```

Comment:
Curious why this one stayed in viewUtils but the switchModel and unloadWindow were moved to utils

Reply:
Anything that was still used by YUI modules had to stay in views/utils.js. In the case of `isRedirectError` it was used by both init code and YUI modules so it remained. As I port more of the views over hopefully I'll be able to move more of these utils.